bsp4j/src/main/java/ch/epfl/scala/bsp4j/BuildServer.java

```diff
@@ -189,5 +189,12 @@ public interface BuildServer {
     @JsonRequest("buildTarget/cleanCache")
     CompletableFuture<CleanCacheResult> buildTargetCleanCache(CleanCacheParams params);
 
+    /**
+     * Notification sent from the client to the server when the user wants to send
+     * input to the stdin of the running target.
+     */
+    @JsonNotification("run/readStdin")
+    void onRunReadStdin(PrintParams params);
```

Comment:
For naming consistency, I think it would be better to use `ReadParams` instead of `PrintParams`, at the cost of duplication.